Cards/CardBase.gd

```diff
@@ -31,16 +31,17 @@ func parse_card_data(card_data: Dictionary) -> void:
 	# TODO
 	pass
 
-func _apply_all_effects(target: Entity) -> void:
+func _apply_all_effects(caster: Entity, target: Entity) -> void:
 	for effect_data: EffectData in card_effects_data:
-		effect_data.apply_effect_data(target)
+		effect_data.apply_effect_data(caster, target)
 
 
 func can_play_card(caster: Entity, target: Entity) -> bool:
 	return caster.get_party_component().can_play_on_entity(application_type, target)
 
 
 func on_card_play(caster: Entity, target: Entity) -> void:
-	_apply_all_effects(target)
+	_apply_all_effects(caster, target)
+	# TODO add caster as parameter to use stats
```

Comment:
Is this TODO still needed? Looks like Caster is being passed in.

Reply:
not needed anymore, I forgot to remove the TODO